Logic/item_classification.py

```diff
@@ -0,0 +1,90 @@
+'''Item Classification Module'''
+
+import cv2
+import math
+
+import Config.eight_ball_lookup as lookup
+
+class ItemClassification:
+    '''Responsible for classifying game balls'''
+
+    @staticmethod
+    def get_ball_radius(x_position, y_position):
+        '''Calculate radius from ball point'''
+
+        return math.sqrt(math.pow(lookup.BALL_RADIUS - x_position, 2) + math.pow(lookup.BALL_RADIUS - y_position, 2))
+
+    def get_ball_pixels(self, frame, position):
+        '''Responsible for returning an array of pixels that represent the circle'''
+
+        ball_pixels = []
+
+        min_x_position = position[0] - lookup.BALL_RADIUS
+        min_y_position = position[1] - lookup.BALL_RADIUS
+        max_x_position = position[0] + lookup.BALL_RADIUS
+        max_y_position = position[1] + lookup.BALL_RADIUS
+
+        ball_frame = frame[min_y_position:max_y_position, min_x_position:max_x_position].copy()
+
+        for x_position, _ in enumerate(ball_frame[0:-1]):
+            for y_position, _ in enumerate(ball_frame[0:-1]):
+                if self.get_ball_radius(x_position, y_position) < lookup.BALL_RADIUS:
+                    ball_pixels.append(ball_frame[x_position][y_position])
+
+        return ball_pixels
+
+    @staticmethod
+    def get_white_count(ball_pixels):
+        '''Finding the number of white pixels within the ball pixels'''
+
+        white_count = 0
+
+        for pixel in ball_pixels:
+            is_r_valid = 192 <= pixel[0] <= 255
+            is_g_valid = 192 <= pixel[1] <= 255
+            is_b_valid = 192 <= pixel[2] <= 255
+
```

Comment:
The values [0], [1] and [2] correspond to BGR not RGB. Just swap is_r_valid with is_b_valid. But essentially no difference.

Reply:
Done